napari/layers/_points_layer/model.py

```diff
@@ -426,6 +431,32 @@ def get_message(self, coord, value):
             msg = msg + ', index ' + str(value)
         return msg
 
+    def _update_thumbnail(self):
+        """Update thumbnail with current points and colors.
+        """
+        colormapped = np.zeros(self._thumbnail_shape)
+        colormapped[..., 3] = 1
+        if len(self._points_view) > 0:
+            min_vals = [self.range[-2][0], self.range[-1][0]]
+            shape = np.ceil(
+                [
+                    self.range[-2][1] - self.range[-2][0] + 1,
+                    self.range[-1][1] - self.range[-1][0] + 1,
+                ]
+            ).astype(int)
+            zoom_factor = np.divide(self._thumbnail_shape[:2], shape).min()
+            coords = np.floor(
+                (self._points_view - min_vals + 0.5) * zoom_factor
+            ).astype(int)
+            coords = np.clip(
+                coords, 0, np.subtract(self._thumbnail_shape[:2], 1)
+            )
+            for c in coords:
+                colormapped[c[0], c[1], :] = Color(self.face_color).rgba
+        colormapped[..., 3] *= self.opacity
+        colormapped = img_as_ubyte(colormapped)
```

Comment:
I think we should suppress the precision loss warnings for now until skimage 0.16 is released (#349).

Reply:
agreed - fine to do in a followup PR - I see you just made #355